sdk/core/azure-core/CMakeLists.txt

```diff
@@ -15,7 +15,7 @@ find_package(Threads REQUIRED)
 # min version for `CURLSSLOPT_NO_REVOKE`
 # https://curl.haxx.se/libcurl/c/CURLOPT_SSL_OPTIONS.html
 set(CURL_MIN_REQUIRED_VERSION 7.44)
-find_package(CURL ${CURL_MIN_REQUIRED_VERSION} CONFIG)
+find_package(CURL ${CURL_MIN_REQUIRED_VERSION} CONFIG QUIET)
```

Comment:
Why are we doing this? What's the harm in keeping the warning visible?

Reply:
Victor recommended it, so yes we remove the warning, but the very next line ("REQUIRED") fails, no need to have an additional warning then.